server/fishtest/util.py

```diff
@@ -343,17 +343,18 @@ def is_active_sprt_ltc(run):
 
 
 def remaining_hours(run):
+    if "spsa" in run["args"]: # Tunes are frequently stopped short, and with "autoselect" may have too many games anyways
+        return 0
     r = run["results"]
+    N = sum(r["pentanomial"]) if "pentanomial" in r else r["wins"] + r["losses"] + r["draws"]
```

Comment:
Check if this is the intent:
 - a run with "pentanomial" field has "wins", "losses", "draws" fields as well
 - `sum(r["pentanomial"])` is half the number of games played

Reply:
ah, i was borrowing from Mineta's code lol, and did not stop to think about it